### ISSUE TITLE
Alpha to 0.5 with l2 penalty, instead of 0

### ISSUE THREAD

**Issue by gaetan-dion** [vertica/VerticaPy#61]
With penalty=ENet, alpha value is modified :   
```python
model = LogisticRegression(name = "schematest.ridge_titanic_JF",cursor = cur,  penalty = "ENet",tol = 1e-4,C = 20.0,max_iter = 100, solver = 'CGD',l1_ratio = 0.1)
````
**Output :**   
```
logistic_reg('schematest.ridge_titanic_JF', 'schematest.titanic_tomodel', '"survived"', '"pclass_0", "pclass_1", "female", "age_1", "age_2", "age_3", "age_4"'
USING PARAMETERS optimizer='cgd', epsilon=0.0001, max_iterations=100, regularization='enet', lambda=20, alpha=0.1)
```

However, it is not normal that alpha be equal to 0.5 when we use L2 penalty (Ridge),  whereas is equal to 1 with L1 (Lasso)

**Output :**   
```
logistic_reg('schematest.ridge_titanic_JF', 'schematest.titanic_tomodel', '"survived"', '"pclass_0", "pclass_1", "female", "age_1", "age_2", "age_3", "age_4"'
USING PARAMETERS optimizer='newton', epsilon=0.0001, max_iterations=100, regularization='l2', lambda=20, alpha=0.5)

logistic_reg('schematest.ridge_titanic_JF', 'schematest.titanic_tomodel', '"survived"', '"pclass_0", "pclass_1", "female", "age_1", "age_2", "age_3", "age_4"'
USING PARAMETERS optimizer='cgd', epsilon=0.0001, max_iterations=100, regularization='l1', lambda=20, alpha=1)
```

**Comment by oualib** [comment]
Hi Gaëtan,

This will be corrected in the next version! Thanks for the inputs

**Comment by oualib** [comment]
The issue is solved in v0.4.1
